eval/run_benchmark.py

```
@@ -0,0 +1,183 @@
+        if result.returncode == 3:
+            print(f"    WARN: {version} tool error on {file_path.name}: {result.stderr.strip()[:200]}", file=sys.stderr)
+            return []
+        try:
+            return json.loads(result.stdout)
+        except json.JSONDecodeError:
+            print(f"    WARN: {version} non-JSON output on {file_path.name}", file=sys.stderr)
+            return []
+    finally:
+        shutil.rmtree(quorum_home, ignore_errors=True)
+
+def run_pal(file_path: Path, rel_path: str) -> list[dict]:
```

Ruff (0.15.12)

[warning] Unused function argument: `file_path`

(ARG001)